Apps/Src/ReadTritium.c

```diff
@@ -150,7 +167,7 @@ static void assertTritiumError(tritium_error_code_t motor_err){
 	Error_ReadTritium = (error_code_t)motor_err; // Store error code for inspection info
 	if(motor_err == T_NONE) return; // No error, return
 	
-	if(motor_err != T_HALL_SENSOR_ERR){
+	if(motor_err != T_HALL_SENSOR_ERR && motor_err != T_WATCHDOG_LAST_RESET_ERR){
```

Comment:
I think it makes sense to only try to restart the motor a set number of times instead of limiting it by type of event. I'll rename some variables to make that clearer